regions/shapes/ellipse.py

```diff
@@ -102,16 +102,27 @@ def to_mask(self, mode='center', subpixels=5):
         else:
             use_exact = 1
 
-        fraction = elliptical_overlap_grid(xmin, xmax, ymin, ymax, nx, ny,
-                                           self.major, self.minor,
-                                           self.angle.to(u.deg).value,
-                                           use_exact, subpixels)
+        fraction = elliptical_overlap_grid(
+            xmin, xmax, ymin, ymax, nx, ny,
+            self.major, self.minor,
+            self.angle.to(u.deg).value,
+            use_exact, subpixels,
+        )
 
         return Mask(fraction, bbox=bbox)
 
     def as_patch(self, **kwargs):
-        # TODO: needs to be implemented
-        raise NotImplementedError
+        """Matplotlib patch object for this region (`matplotlib.patches.Ellipse`).
+        """
+        from matplotlib.patches import Ellipse
+        xy = self.center.x, self.center.y
+        width = 2 * self.major
+        height = 2 * self.minor
+        # TODO: think about how to map major / minor / angle to MPL parameters.
```

Comment:
I'm not sure where you changed the major/minor order, but what you have here is correct for the ellipse.  Photutils defines the `theta` as the counterclockwise rotation of the semimajor axis from the `x` axis.  This is the same for the `EllipsePixelRegion`.

https://github.com/cdeil/regions/blob/387c5a60a699bffdf3546c620cf6396fd5a9d06c/regions/shapes/ellipse.py#L25